content/en/docs/general-usage/sdv-introduction/_index.md

```diff
@@ -77,18 +76,19 @@ A good starting point to figure out what signals are available is the website of
 
 ![digital.auto Vehicle API](digitalauto-vehicle-api.png)
 
-In the next step, one can ideate on new applications, start coding, and get fast feedback using the prototyping feature of the playground.
-Once one is confident with the initial version, one can continue the development journey with the help of [Eclipse Velocitas](https://eclipse.dev/velocitas/), 
+In the next step, we can ideate on new applications, start coding, and get fast feedback using the prototyping feature of the playground.
+Once we are confident with the initial version, we can continue the development journey with the help of [Eclipse Velocitas](https://eclipse.dev/velocitas/), 
 which comes with several features making the life of vehicle application developers easier.
 
 ![digital.auto Prototyping](digitalauto-prototype.png)
 
 ### Eclipse Velocitas
+[Eclipse Velocitas™](https://eclipse.dev/velocitas/docs/about/#product) is an open source project providing an end-to-end, scalable and modular development tool chain to create containerized Vehicle Apps, offering a comfortable, fast and efficient development experience to increase the speed of a development team (velocity).
 
-As a starting point, Eclipse Velocitas provides a [template repository](https://github.com/eclipse-velocitas/vehicle-app-python-template).
-To kick-start the development of a new application, one can create an individual repository from this template.
-As an alternative, Eclipse Velocitas provides a CLI application with which one can configure the creation of a tailored vehicle application repository.
-Either way, the resulting repository containers [skeleton code for the actual application](https://github.com/eclipse-velocitas/vehicle-app-python-template/blob/main/app/src/main.py)
+Among other things, Eclipse Velocitas provides a [template repository](https://github.com/eclipse-velocitas/vehicle-app-python-template).
+To kick-start the development of a new application, we can create an individual repository from this template.
```

Comment:
Would use 'you' instead of 'we', see my other comment.

Reply:
dito